src/swsssdk/configdb.py

```diff
@@ -49,19 +47,26 @@ def __init__(self, decode_responses=True, **kwargs):
            'namespace' is implicitly passed to the parent SonicV2Connector class.
         """
         super(ConfigDBConnector, self).__init__(**kwargs)
+        # Trick: to achieve static/instance method "overload", we must use initize the function in ctor
+        # ref: https://stackoverflow.com/a/28766809/2514803
+        self.serialize_key = self._serialize_key
+        self.deserialize_key = self._deserialize_key
+
+        self.TABLE_NAME_SEPARATOR = '|'
+        self.KEY_SEPARATOR = '|'
         self.handlers = {}
 
     def __wait_for_db_init(self):
         client = self.get_redis_client(self.db_name)
         pubsub = client.pubsub()
-        initialized = client.get(self.INIT_INDICATOR)
+        initialized = client.get(ConfigDBConnector.INIT_INDICATOR)
```

Comment:
It appears to use the class value even with self prefix. Effectively a this code change is no-op, though your update appears good. Guess, ConfigDBCOnnector is soon due for retirement, so we may not need to  polish.

Reply:
That's my convention, in order not to misuse class variable.

---
In reply to: [549806293](https://github.com/Azure/sonic-py-swsssdk/pull/99#discussion_r549806293) [](ancestors = 549806293)